INSTALL.md

```diff
@@ -47,4 +47,4 @@ $ cu -s 57600 -l /dev/ttyUSB0
 
 ## Troubleshooting boot issues without a serial port
 
-Serenity is a graphical OS and thus boots directly to the desktop. If your computer doesn't have a serial port and it fails to boot, you can force Serenity to boot into text mode by editing **Kernel/Arch/i386/Boot/boot.S** and removing **| MULTIBOOT_VIDEO_MODE** from the end of the **multiboot_flags** line before (re)running **makeall.sh**.
+Serenity is a graphical OS and thus boots directly to the desktop. If your computer doesn't have a serial port and it fails to boot, you can force Serenity to boot into text mode by editing **Kernel/Arch/i386/Boot/boot.S** and removing **| MULTIBOOT_VIDEO_MODE** from the end of the **multiboot_flags** line before (re)running **make** and **make install**.
```

Comment:
This can just be `make install`.

Reply:
Also, I don't think that this is working: I removed `| MULTIBOOT_VIDEO_MODE` from `Kernel/Arch/i386/Boot/boot.S`, but Serenity still booted directly to the desktop.

Reply:
The section about removing `| MULTIBOOT_VIDEO_MODE` to start in text mode was incorrect even before I submitted this pull request, it isn't new. 

Also, I added a `SerenityOS (text mode)` option to the GRUB configs that use the `boot_mode` option to boot into text mode without having to modify boot.S.